src/Components/ShareProfile.css

```diff
@@ -0,0 +1,17 @@
+.profile__container {
+  display: flex;
```

Comment:
It would be better to stick to prime react css classes than adding all this extra css.

Reply:
oh, I didn't see that we are using prime react. I saw those .css and assumed it would be normal CSS :P

Reply:
Is it okay now?